generate_solve_rllib_test.py

```diff
@@ -547,12 +547,12 @@ def _get_action_space_(self) -> GymSpace[BelugaGymCompatibleDomain.T_event]:
 
     print("Creating Sk{}PDDLDomain".format("P" if args.probabilistic else ""))
     domain_factory = lambda: (
-        SkdPPDDLDomain(inst, problem_name, problem_folder)
+        SkdPPDDLDomain(inst, problem_name, problem_folder, classic=classic)
```

Comment:
This will result in a Python exception since the `SkdPPDDLDomain` does not support numeric encoding. Can you please provide a way to write an error message and to gracefully exit the script when the user selects the numeric encoding and the ppddl model options at the same time? Thanks

Reply:
Yes! I did not notice that as I did not try that PPDDL case yet :)
I added a condition, to show an error and quit. see if that suffice.